remotefile/src/qrmf_proto.cpp

```diff
@@ -63,31 +67,26 @@ int packHeader(char *pDest, int destLimit, quint32 address, bool more_bit)
 int unpackHeader(const char *pBegin, const char *pEnd, quint32 *address, bool *more_bit)
 {
    int retval = 0; //default is to assume that the buffer is too small to parse
-   const char *pNext=pBegin;
    if( (pBegin != 0) && (pEnd != 0) && (address != NULL) && (more_bit != NULL))
    {
-      if (pBegin+1<pEnd)

```

Comment:
In case the message has 0 length which means that pBegin==pEnd then even the first character at pBegin can't be parsed. This check prevented that from happening